v22.2/restore.md

```diff
@@ -159,12 +159,13 @@ Also, a full cluster restore will:
 
 - Restore [temporary tables](temporary-tables.html) to their original database during a full cluster restore.
 - Drop the cluster's `defaultdb` and `postgres` [pre-loaded databases](show-databases.html#preloaded-databases) before the restore begins. You can only restore `defaultdb` and `postgres` if they are present in the original [backup](take-full-and-incremental-backups.html).
+- When the cluster is in a mixed-version state during an [upgrade](upgrade-cockroach-version.html), a full cluster restore will fail.
```

Comment:
Hey @stevendanna I gleaned from the PR that full cluster restores **will** fail, but is that true?

Reply:
Yup!  If you wanted to say more, you can say something like "To perform a full cluster restore, you can finalize the upgrade."